### ISSUE TITLE
this error indicates that your module has parameters that were not used in producing loss

### ISSUE THREAD

**Issue by thenighthunter0** [facebookresearch/detectron2#1386]
when using single gpu :
          we can do 'p.requires_grad = True'
but when using gpus:
            'p.requires_grad = True' will get error
this error indicates that your module has parameters that were not used in producing loss

**Comment by ppwwyyxx** [comment]
If you need help to solve an unexpected issue you observed, please include details following the [issue template](https://github.com/facebookresearch/detectron2/issues/new?template=unexpected-problems-bugs.md).
